aegis-web/yo/app/styles/filebrowser.css

```diff
@@ -151,6 +154,7 @@
   padding: 10px;
   overflow: hidden;
   text-overflow: ellipsis;
+  height: 50%;
```

Comment:
This is also affecting AEGIS tools (maybe other places too). Why do you need to set the height like that?
![image](https://user-images.githubusercontent.com/1640100/55372816-4a38a580-5503-11e9-8280-70c5f9ba2a8e.png)